src/mpyl/schema/run_properties.schema.yml

```diff
@@ -54,8 +54,8 @@ properties:
         properties:
           deploy_target:
             description: The deploy target.
-            type: ["string", "null"]
-            enum: [ null,  'PullRequest', 'PullRequestBase', 'Production' ]
+            type: [ "string", "null" ]
+            enum: [ null,  'PullRequest', 'PullRequestBase', 'Acceptance', 'Production' ]
```

Comment:
@SamTheisens Acceptance was taken out of the list in https://github.com/Vandebron/mpyl/commit/bf7e9f0535b4111288633daaf819976b3db9f133, I assume by accident?

Reply:
I should be more careful with reviewing I guess :D